qiskit/transpiler/passes/layout/vf2_layout.py

```diff
@@ -31,6 +30,7 @@ class VF2Layout(AnalysisPass):
 
         * ``"solution found"``: If a perfect layout was found.
         * ``"nonexistent solution"``: If no perfect layout was found.
+        * ``">2q gates in basis"``: If VF2Layout can't work with basis
```

Comment:
I missed this in the original PR (or perhaps I didn't review), but this feels like it should be an enum type, rather than an arbitrary string, especially since we're starting to dispatch logic based on it.  I'm not sure it's a suggestion for this PR, though.

Reply:
I can change it here, we just merged the pass this morning in #6620 so now is the time to change it

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7213/commits/39ddd9d190a87ecc42db8ebb0b928fef74ac5c23